### PR TITLE
Adding known issues documentation for WPF

### PR DESCRIPTION
**Pull request by miguep**
Adding known issues documentation for WPF

### CODE REVIEW

File: Documentation/KnownIssues/wpf-pma-window-drag.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Per-Monitor DPI aware WPF applications positions mouse cursor incorrectly when dragging between monitors
```
Comment (rpetrusha) [inline Documentation/KnownIssues/wpf-pma-window-drag.md]:
```suggestion
# Per-Monitor DPI-aware WPF applications position mouse cursor incorrectly when dragging between monitors
```

File: Documentation/KnownIssues/wpf-pma-window-drag.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+## Symptoms
+A user running a WPF application that enables the Per-Monitor DPI awareness feature introduced in .NET Framework 4.7.2 may notice that the mouse cursor is in a different location relative to the window after dragging the window from one monitor to another.
+
+## Cause
+This issue is caused by a calculation error when resizing and placing the window after switching DPI.
```
Comment (rpetrusha) [inline Documentation/KnownIssues/wpf-pma-window-drag.md]: Is a hotfix going to be made available? If so, it should be mentioned, as it is in #990, for example.

Comment (miguep) [reply Documentation/KnownIssues/wpf-pma-window-drag.md]: @vatsan-madhavan what's the plan for this bug?

Comment (vatsan-madhavan) [reply Documentation/KnownIssues/wpf-pma-window-drag.md]: We don’t plan on fixing this in .net framework. It will be addressed in .net core.

**Comment by rpetrusha** [comment]
I'll merge your PR now, @miguep.

**Pull request merged**